labgrid/driver/shelldriver.py

```diff
@@ -364,10 +365,8 @@ def _target_cleanup(tmpfile):
         stream = io.BytesIO(buf)
 
         # We first write to a temp file, which we'll `dd` onto the destination file later
-        tmpfile = self._run_check('mktemp')
-        if not tmpfile:
-            raise ExecutionError('Could not make temporary file on target')
-        tmpfile = tmpfile[0]
+        tmpfile = f'/tmp/tmp-{uuid4()}'
+        self._run_check(f'touch {tmpfile}')
```

Comment:
How about only doing this as a workaround when mktemp fails? Then we can keep using the normal approach for most cases?

Reply:
That would work for me - as a fallback. What are your thoughts about it @jluebbe ?